### PR TITLE
Revert Mat 9679 space around character length

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## MADiE PR

Jira Ticket: [MAT-9679](https://jira.cms.gov/browse/MAT-9679)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).
* [ ] All CDN/Web dependencies are hosted internally (i.e MADiE-Root Repo).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

**Comment by sb-cecilialiu** [comment]
This is revert the previous change for MAT-9679 (https://github.com/MeasureAuthoringTool/madie-design-system/pull/391/changes/bdb1f2db3ba45b2c43477699446f3b1f1e1c65aa)

File: react/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "@madie/madie-design-system",
-    "version": "1.2.86",
+    "version": "1.2.85",
```
Comment (chubert-sb) [inline react/package.json]: I think you might have to make this 1.2.87 or the newest version will still have the change

Comment (sb-cecilialiu) [inline react/package.json:3]: @chubert-sb So I used 1.2.86 for the change, but 1.2.86 didn't get published. This morning I tried 1.2.87 hoping to get it published. I'm reverting it back to 1.2.85 before the change. I think it should be 1.2.85?

**Review by chubert-sb — DISMISSED** [review]

**Review by chubert-sb — APPROVED** [review]
